libkbfs/keybase_service_base.go

```diff
@@ -619,3 +619,25 @@ func (k *KeybaseServiceBase) GetTLFCryptKeys(ctx context.Context,
 
 	return res, nil
 }
+
+// GetTLFCanonicalID implements the TlfKeysInterface interface for
+// KeybaseServiceBase.
+func (k *KeybaseServiceBase) GetTLFCanonicalID(ctx context.Context,
+	arg keybase1.GetTLFCanonicalIDArg) (res keybase1.TLFCanonicalID, err error) {
+	ctx = ctxWithRandomIDReplayable(ctx, CtxKeybaseServiceIDKey, CtxKeybaseServiceOpID,
+		k.log)
+	tlfHandle, err := k.getHandleFromFolderName(ctx, arg.TlfName, arg.Public)
```

Comment:
I don't find the `GetTLFCryptKeys` name to be specific.  It sounds very generic to me and doesn't even mention the TLF ID that you are concerned about.